frontend/server/src/Controllers/User.php

```diff
@@ -3640,17 +3643,39 @@ public static function getProfileDetailsForSmarty(\OmegaUp\Request $r) {
                     'Identity'
                 );
             }
-            $smartyProperties = [
-                'profile' => self::getProfileDetails($r->identity, $identity),
-            ];
+            $profile =  self::getProfileDetails($r->identity, $identity);
+            $contests = self::apiContestStats($r);
+            $solvedProblems = self::apiProblemsSolved($r);
+            $unsolvedProblems = self::apiListUnsolvedProblems($r);
+            $createdProblems = self::apiProblemsCreated($r);
+            $stats = self::apiStats($r);
+            $badges = \OmegaUp\Controllers\Badge::apiList($r);
+            $ownedBadges = \OmegaUp\Controllers\Badge::apiMyList($r);
+            //print_r($stats);
+            //print_r($profile);
+            //print_r($unsolvedProblems);
+            //print_r($solvedProblems);
+            //print_r($createdProblems['problems']);
         } catch (\OmegaUp\Exceptions\ApiException $e) {
-            $smartyProperties = [
+            $profile = [
```

Comment:
En vez de `$profile` podrías agregar `$payload` para que quede de la siguiente forma:
```suggestion
            $payload = [
                'statusError' => $e->getErrorMessage(),
            ];
```
Para que funcione tendrías que declarar `$payload = [];` antes de L3636